static/app/views/settings/account/accountSecurity/index.tsx

```diff
@@ -198,6 +199,23 @@ class AccountSecurity extends AsyncView<Props> {
                             </RemoveConfirm>
                           </Tooltip>
                         )}
+
+                        {!isBackupInterface && isEnrolled && allowRotationInPlace && (
+                          <Tooltip
```

Comment:
You can drop the `Tooltip` component and use `title` on the button

Reply:
Thanks. Would the same also apply to the 'delete' button above (from which I cribbed), or is that different because of the way it's wrapped or disabled?

Reply:
We could actually remove it there if we pass the disabled state through this prop https://github.com/getsentry/sentry/blob/c5c74bb8ef262f4f11783b5252c44a75c2df9837/static/app/components/button.tsx#L32